var/spack/repos/builtin/packages/tandem/package.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class Tandem(CMakePackage):
+    """Tandem is a scientific software for SEAS modelling and for solving Poisson
+    and linear elasticity problems. It implements the Symmetric Interior Penalty
+    Galerkin (SIPG) method using unstructured simplicial meshes (triangle meshes
+    in 2D, tetrahedral meshes in 3D)."""
+
+    homepage = "https://tandem.readthedocs.io/en/latest/"
+    version("main", git="https://github.com/TEAR-ERC/tandem.git", branch="main", submodules=True)
+
+    # we cannot use the tar.gz file because it does not contains submodules
+    version("1.0", git="https://github.com/TEAR-ERC/tandem.git", tag="v1.0", submodules=True)
+    patch("fix_v1.0_compilation.diff", when="@1.0")
+
+    maintainers = ["dmay23", "Thomas-Ulrich"]
```

Comment:
@dmay23 Are you okay with being listed as a maintainer for this package?

(See https://spack.readthedocs.io/en/latest/packaging_guide.html#maintainers.)

Reply:
@tldahlgren  Yes - no problem to be a maintainer.